index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <div id="img-div">
+        <img id="image" src="https://upload.wikimedia.org/wikipedia/commons/b/ba/Norman_Borlaug.jpg" alt="">
+        <h2 id="img-caption">Dr. Borlaug giving a conference</h2>
+    </div>
+    <p id="tribute-info">
+      Norman Ernest Borlaug (Cresco, Iowa, Estados Unidos, 25 de marzo de 1914-Dallas, Texas, Estados Unidos, 12 de septiembre de 2009), citado como Norman Burlaug, fue un ingeniero forestal, genetista, fitopatólogo, humanista estadounidense, considerado por muchos el padre de la agricultura moderna y de la revolución verde, ha sido llamado "el hombre que salvó mil millones de vidas". Sus esfuerzos en los años 1960 para introducir las semillas híbridas a la producción agrícola en México, Pakistán e India provocaron un incremento notable de la productividad agrícola, y algunos lo consideran responsable de haber salvado más de 1000 millones de vidas humanas.
+    </p>
+    <a id="tribute-link" href="https://es.wikipedia.org/wiki/Norman_Borlaug" target="_blank">more info...</a>
+    </main>
```

Comment:
I think you did a great job here, however, there are some things to improve here. :smiley: 

- This block of code should be inside a section tag for clarity. Kindly wrap your landing page section inside `section` tags.  We do this because as you developed bigger projects, clarity is a vital skill to have as a developer. With good code clarity, your colleagues will be able to easily review your changes and identify what you add to each separate section.
![Section division](https://user-images.githubusercontent.com/90111138/199050583-544d0035-7140-41ad-be6f-cd50d516a733.png)

- Also in this block of code your inside tags should have a space indentation for clarity. This is a simple way that coders use to know what is inside the tag. Kindly correct the spacing of your tags to be similar to the following example.
![Tag indentation](https://user-images.githubusercontent.com/90111138/199051694-c0d5b68d-dac5-4c4f-9d6d-92500e7b499c.png)
As you can see in this example with the correct indentation spacing you can easily see what is inside each tag.